pipeline/metadata/6.0.yaml

```diff
@@ -575,6 +575,16 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
           metadata:
             - rgw
+        - name: "tier-1_rgw_ms_archive_test-upgrade-5-to-latest"
+          execution_time: "3h 19m 29s"
+          suite: "suites/quincy/rgw/tier-1_rgw_ms_archive_test-upgrade-5-to-latest.yaml"
+          global-conf: "conf/quincy/rgw/rgw_ms_archive.yaml"
```

Comment:
Looks like this conf file not present in quincy..Could you please confirm
if it is missed to include in the PR, please include it

Reply:
@ckulal , thank you have included the pacific path for the conf file.